### DISCUSSION TITLE
associate the mobile phone number to the customer instead of the address

### DISCUSSION THREAD

**Discussion by jf-viguier** [PrestaShop/PrestaShop#37221] [Ideas]
### The problem

The phone_mobile is actually associated to Address entity but no more asked in front.

For PrestaShop 1.6 and less, maybe, taking the last mobile number and add it.

### The solution

The idea is to add a phone_mobile field to customer entity (and customer form).
We have the phone field on the address pour delivery and fixed phones.

### Do you plan to work on this subject?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [ ] I'm willing to submit a Pull Request that implements this feature.
- [ ] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by Maofree** [comment]
Hi
I have a client who asked me the possibility to log in using the phone number or email. Even with the phone they have a way to contact the client for promotions on whatsapp like in this website

![1](https://github.com/user-attachments/assets/1ecae4d3-b490-4059-a208-255780047104)

bye
